Sync/KeysPayload.swift

```diff
@@ -3,6 +3,7 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import Foundation
+import Account
```

Comment:
I guess you're just moving the `Keys`/`Payload` stuffs from `Sync` to `Account`?

Reply:
Yep. Essentially, I've moved all the crypto stuff from `Sync` to `Account` for two reasons. One, `Sync` essentially requires `Account` anyway, so it makes no difference from a `Sync` standpoint. Two, eventually `Sync` will not be required in NotificationService for send tab to work once support for old clients is dropped. At that point, NotificationService will only need to link to the `Account` framework which will have everything we need without having to import anything else.

Reply:
👍 +1: